apps/web/src/app/next_api/revalidate-global/route.ts

```diff
@@ -0,0 +1,38 @@
+import { revalidateTag } from "next/cache";
+import type { NextRequest } from "next/server";
+import { NextResponse } from "next/server";
+
+// this endpoint will revalidate a page by tag or path
+// this is to achieve on-demand revalidation of pages that use this data
+// send either `collection` and `slug` or `revalidatePath` as query params
+export function GET(request: NextRequest): NextResponse {
```

Comment:
This should probably be a POST endpoint to be semantic since it performs side effects :nerd_face: